frontend/hub/my-imports/MyImports.tsx

```diff
@@ -0,0 +1,235 @@
+import { useTranslation } from 'react-i18next';
+import { useEffect, useState } from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+import { useGet } from '../../common/crud/useGet';
+import { HubItemsResponse } from '../common/useHubView';
+import { hubAPI } from '../common/api/formatPath';
+import { HubRoute } from '../main/HubRoutes';
+import {
+  Scrollable,
+  PageLayout,
+  PageHeader,
+  useGetPageUrl,
+  IFilterState,
+} from '../../../framework';
+import {
+  PageSection,
+  Drawer,
+  DrawerContent,
+  DrawerContentBody,
+  Flex,
+  FlexItem,
+  DrawerPanelContent,
+  DrawerHead,
+  DrawerPanelBody,
+  Title,
+} from '@patternfly/react-core';
+import { CollectionImport, CollectionVersionSearch } from '../collections/Collection';
+import { ImportLog } from './components/ImportLog';
+import { ImportList } from './components/ImportList';
+
+export function MyImports() {
+  const { t } = useTranslation();
+  const getPageUrl = useGetPageUrl();
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const namespaceQP = searchParams.get('namespace') ?? '';
+  const nameQP = searchParams.get('name') ?? undefined;
+  const statusQP = searchParams.get('status') ?? undefined;
+  const versionQP = searchParams.get('version') ?? undefined;
+  const pageQP = Number(searchParams.get('page') ?? 1);
+  const perPageQP = Number(searchParams.get('perPage') ?? 10);
```

Comment:
might want to use `||` instead of `??` here,

`Number("foo")` returns `NaN`, `parseInt("foo")` returns `0`, neither is a valid value.